Snowball-Finance/axial-frontend
# rewards claim UI integration on rewards page

To test:
1. Check rewards tokens are present in the rewards pool list
2. Check rewards info is in deposit and withdraw page
3. Check the claim rewards functionality

Need to implement:
Haven't tested the functionality
